pkg/networkservice/common/localbypass/server_test.go

```diff
@@ -23,6 +23,8 @@ import (
 	"net/url"
 	"testing"
 
+	localbypass_reg "github.com/networkservicemesh/sdk/pkg/registry/common/localbypass"
```

Comment:
I would suggest this name
```suggestion
	registrylocalbypass "github.com/networkservicemesh/sdk/pkg/registry/common/localbypass"
```

Reply:
Thanks.